keylime/cloud_verifier_tornado.py

```diff
@@ -278,7 +278,7 @@ def get(self):
             try:
                 agent = (
                     session.query(VerfierMain)
-                    .options(joinedload(VerfierMain.ima_policy))
```

Comment:
If we aren't using any of the fields in the `ima_policy` joined table then why do a join at all?

Reply:
Right we are no longer using `joinedload` there